CHANGELOG.md

```diff
@@ -4,6 +4,74 @@
+- `pkg/ottl`: Add SliceToMap function (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35256)
+
+### 💡 Enhancements 💡
+
+- `mdatagen`: Add otlp as supported distribution (https://github.com/open-telemetry/opentelemetry-collector/pull/11527)
+- `batchprocessor`: Move single shard batcher creation to the constructor (https://github.com/open-telemetry/opentelemetry-collector/pull/11594)
+- `service`: add support for using the otelzap bridge and emit logs using the OTel Go SDK (https://github.com/open-telemetry/opentelemetry-collector/pull/10544)
+
+- `hostmetricsreceiver`: Add the system.uptime metric in the hostmetrics receiver (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31627)
+  This metric is provided by the new `system` scraper.
+
+### 🧰 Bug fixes 🧰
+
+- `service`: ensure traces and logs emitted by the otel go SDK use the same resource information (https://github.com/open-telemetry/opentelemetry-collector/pull/11578)
+- `config/configgrpc`: Patch for bug in the grpc-go NewClient that makes the way the hostname is resolved incompatible with the way proxy setting are applied. (https://github.com/open-telemetry/opentelemetry-collector/pull/11537)
+- `builder`: Update builder default providers to lastest stable releases (https://github.com/open-telemetry/opentelemetry-collector/pull/11566)
+
+- `pkg/stanza`: Fixed bug causing Operators with DropOnErrorQuiet to send log entries to the next operator. (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35010)
+  This issue was introduced by a bug fix meant to ensure Silent Operators are not logging errors (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35010). With this fix,
+  this side effect bug has been resolved.
+- `receiver/filelog`: fix record counting with header (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35869)
+- `pkg/stanza/operator`: Retain Operator should propagate the severity field (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35832)
+  The retain operator should propagate the severity field like it does with timestamps.
+
+- `pkg/stanza`: Handle error of callback function of `ParserOperator.ProcessWithCallback` (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35769)
+  `ProcessWithCallback` of `ParserOperator` first calls the `ParseWith` method
+  which properly handles errors with `HandleEntryError`.
+  Then the callback function is called and its returned error should also
+  be handled by the `HandleEntryError` ensuring a consistent experience.
+---
+
+</details>
+
+#### Dynatrace distribution changelog:
+
+### 💡 Enhancements 💡
+
+- `transformprocessor`: Extend the user data filtering example with a statement for masking the `client.address` attribute. (#356)
+- `transformprocessor`: Extend the credit card filtering example with log body filtering (#357)
```

Comment:
[nitpick] I forgot to add that also IBAN log body filtering is included here

Reply:
thanks, I have adapted the not to also reflect that